docs/cardinal/game/configuration/cardinal.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: 'Cardinal'
+description: 'This section configures Cardinal, which handles shard routing, logging, and rollup modes.'
+---
+
+```
+[cardinal]
+BASE_SHARD_ROUTER_KEY = string
+BASE_SHARD_SEQUENCER_ADDRESS = string
+CARDINAL_LOG_LEVEL = string
+CARDINAL_LOG_PRETTY = boolean
+CARDINAL_NAMESPACE = string
+CARDINAL_ROLLUP_ENABLED = boolean
+REDIS_ADDRESS = string
+REDIS_PASSWORD = string
+TELEMETRY_PROFILER_ENABLED = boolean
+TELEMETRY_TRACE_ENABLED = boolean
+```
+
+### BASE_SHARD_ROUTER_KEY
+
+A secure authentication token used for routing game shards. It ensures proper routing within the Cardinal system.
+
+**Importance** : This value must be kept secure, as it is critical for shard operations.
+
+**Example**
+```
+BASE_SHARD_ROUTER_KEY = 'e99e9ed8d90e593ec8ef86d6e9cbeb0de5aabfa415d2fd369d6ee1974dc2bb7d'
+```
+
+### BASE_SHARD_SEQUENCER_ADDRESS
+
+The address of the sequencer service, which coordinates shard operations. If rollup mode is enabled, this address points to the sequencer handling transactions.
+
+**Importance** : Required if Cardinal’s rollup mode is enabled.
+
+**Example**
+```
+BASE_SHARD_SEQUENCER_ADDRESS = 'localhost:9601'
+```
+
+### CARDINAL_LOG_LEVEL
+
+Sets the verbosity level of logging in Cardinal. The available levels are debug, info, warn, error, fatal, panic, disabled, and trace.
+
+**Importance** : Determines the detail of logs generated, with info being the most commonly used level in production environments.
+
+**Example**
+```
+CARDINAL_LOG_LEVEL = 'info'
+```
+
+### CARDINAL_LOG_PRETTY
+
+When set to true, enables human-readable (pretty) logs for easier debugging. By default, it’s disabled to avoid a performance penalty.
+
+**Importance** : Use for development and debugging, but disable in production for better performance.
+
+**Example**
+```
+CARDINAL_LOG_PRETTY = true
+```
+
+### CARDINAL_NAMESPACE
+
+A unique identifier for the Cardinal shard namespace. This ensures that different shards don’t interfere with each other and prevents signature replay attacks.
+
+**Importance** : Must be unique per shard to maintain system security.
+
+**Example**
+```
+CARDINAL_NAMESPACE = 'defaultnamespace'
+```
+
+### CARDINAL_ROLLUP_ENABLED
+
+A unique identifier for the Cardinal shard namespace. This ensures that different shards don’t interfere with each other and prevents signature replay attacks.
```

Comment:
This is the description for `CARDINAL_NAMESPACE`

Reply:
ooppss, updated